### ISSUE TITLE
Configuration file read in but not used

### ISSUE THREAD

**Issue by paul-morriss-cko** [SwatKat1977/TApic#6]
In Program.py the configuration file is read in from the command line:

```
    argsParser = argparse.ArgumentParser()

    # 'Optional Arguments' : 'config'.
    argsParser.add_argument('-c', '--configFile', help='Configuration file.',
        default='config.json')

    # Parse the arguments.
    args = argsParser.parse_args()

    # Check if the specified config is valid.
    if os.path.isfile(args.configFile) is False:
        logger.error(f"Configuratiom file '{args.configFile}' isn't valid.")
        sys.exit(2)
```

When it comes to using the config file the default value of config.json is passed in:
`configEntry = configManager.ParseConfigFile('./config.json')`

**Comment by SwatKat1977** [comment]
Line changed in CodeTidy branch:
`configEntry = configManager.ParseConfigFile(args.configFile)`

**Comment by SwatKat1977** [comment]
Fixed in V0.3.0
